CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # `chainweb-node` Changelog
 
+## 2.3 (2020-11-11)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-11-19T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2020-12-22T00:00:00Z**.
```

Comment:
@mightybyte @slpopejoy This is just a placeholder. The final value must be still filled in.